src/order.rs

```diff
@@ -194,6 +194,93 @@ impl Order {
         Ok(&self.state)
     }
 
+    /// Poll the order with timeout until in a final state
+    ///
+    /// Refresh the order state from the server with a timeout `timeout`.
+    /// If a server sets the Retry-After header for rate-limiting access to the CA, the
```

Comment:
nit: backticks for `Retry-After`
```suggestion
    /// If a server sets the `Retry-After` header for rate-limiting access to the CA, the
```

Reply:
changed